xml/snmp_sysdescr.xml

```diff
@@ -2152,11 +2152,11 @@ Copyright (c) 1995-2005 by Cisco Systems
 
   <fingerprint pattern="^(CX\S+) - Flare (\d\S+)$">
     <description>EMC Clariion NAS</description>
-    <example>CX3-10c - Flare 3.26.0.10.5.032</example>
-    <example>CX300 - Flare 2.16.0.300.5.012</example>
-    <example>CX400 - Flare 2.19.0.400.5.040</example>
-    <example>CX500 - Flare 2.07.0.500.5.020</example>
-    <example>CX700 - Flare 2.24.0.700.5.022</example>
+    <example os.product="CX3-10c" os.version="3.26.0.10.5.032">CX3-10c - Flare 3.26.0.10.5.032</example>
```

Comment:
Marking this for follow up PR: Should be `hw.product`

Ugh, sooooooo much of this.